Makefile

```diff
@@ -372,27 +372,28 @@ build/species/%/epitopes.tsv: build/iedb/peptide.tsv build/species/%/taxa.txt
 build/species/%/sources.tsv: build/iedb/peptide_source.tsv build/species/%/taxa.txt
 	qsv search --select 'Organism ID' `cat build/species/$*/taxa.txt` $< --output $@
 
-# TODO!!!
-build/species/%/proteome.tsv: build/species/%/epitopes.tsv build/species/%/sources.tsv
-	protein_tree/protein_tree/select_proteome.py -t $*
+.PRECIOUS: build/species/%/epitopes.tsv build/species/%/sources.tsv
 
-# Use Dengue as an example.
-.PHONY: dengue
-dengue: build/species/12637/proteome.tsv
+build/species/%/proteome.fasta: build/species/%/epitopes.tsv build/species/%/sources.tsv
+	src/protein_tree/src/select_proteome.py -b build/ -t $*
+
+TAXON_IDS := $(shell awk 'NR>1 {print $$2}' build/arborist/active-species.tsv)
```

Comment:
This fails if the `build/arborist/active-species.tsv` does not exist, e.g. after `make clean`.

Reply:
The `:=` means that the value is assigned to the variable immediately when Make loads. It might work using `=`, which will assign only when the variable is used.

Reply:
Fixed.